docs/makefile.md

```diff
@@ -0,0 +1,202 @@
+# Use of the EBS CSI Driver `Makefile`
+
+The EBS CSI Driver comes with a Makefile that can be used to develop, build, test, and release the driver. This file documents the available targets intended for use by humans, the parameters they support, and common usage scenarios.
```

Comment:
np suggest the following: _The EBS CSI Driver comes with a Makefile that can be used to develop, build, test, and release the driver. This file documents the available targets ~intended for use by humans~, the parameters they support, and common usage scenarios._

Reply:
Fixed in next version